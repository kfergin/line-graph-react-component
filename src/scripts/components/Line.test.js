import React from 'react';
import renderer from 'react-test-renderer';

import Line from './Line';

describe('<Line/>', () => {
	const props = {
		points: [
			{x: 0, y: 2},
			{x: 0, y: 1},
			{x: 0, y: 5}
		],
		color: '#00F',
		enterPoint: null,
		shiftX: 10
	};
	it('should render a polyline', () => {
		const tree = renderer.create(
			<Line {...props}/>
		).toJSON();
		expect(tree).toMatchSnapshot();
	});
	// how would you test functions handed to TransitionMotion?
});