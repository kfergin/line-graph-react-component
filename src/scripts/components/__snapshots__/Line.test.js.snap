// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Line/> should render a polyline 1`] = `
<polyline
  fill="none"
  points="0 2 0 1 0 5"
  stroke="#00F"
  strokeLinecap="round"
  strokeLinejoin="round"
  transform="translate(10)"
/>
`;
